kem/schemes/schemes.go

```diff
@@ -20,6 +21,7 @@ var allSchemes = [...]kem.Scheme{
 	sikep434.Scheme(),
 	sikep503.Scheme(),
 	sikep751.Scheme(),
+	hybrid.KyberSike(), // XXX temporary for testing
```

Comment:
In a side note, I noticed that the documentation doesn't shows what is registered, only if someone opens the source file.

Reply:
I don't understand what you mean.

Reply:
run godoc command and see the documentation of scheme package.

Reply:
It's because `allSchemes` is private.  We could add a testable example for `All()` which prints the names of all schemes.  That would show up in the online documentation (but `go doc` doesn't seem to list examples.)

Reply:
I suggest to add a list in the documentation showing schemes. I can include this in the other PR.

go doc  != godoc

Reply:
Then a testable example which outputs the list would be better because that'll force it to keep in synch, whereas a simple list will get out of date.